certified-connectors/iMIS/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "CSI API Key",
+          "description": "Please enter your CSI API Key, found in the CSI Cloud Dashboard.",
+          "tooltip": "If you need an API Key or are experiencing issues, please contact CSI Support (www.csiinc.com/support).",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FCFCFC",
+    "capabilities": [],
+    "publisher": "Computer System Innovations, Inc.",
+    "stackOwner": "Advanced Solutions International, Inc."

```

Comment:
@csijake can you please confirm if you own the API? Why the publisher is different than the stackOwner?

Reply:
@Amjed-Ayoub In short, we have been authorized to publish this connector on behalf of the stack owner. This information is correct. I will send you an email with additional information.